src/Console/Application.php

```diff
@@ -100,6 +96,17 @@ public function run(InputInterface $input = null, OutputInterface $output = null
         return parent::run($input, $output);
     }
 
+    private function logRunningWithDebugger(InputInterface $input)
+    {
+        if (!$input->hasParameterOption(['--quiet', '-q'], true)) {
```

Comment:
Shouldn't it be like this instead, what do you think?
```php
    public function logRunningWithDebugger(string $debugger)
    {
        // --- $this->io->writeln(sprintf(self::RUNNING_WITH_DEBUGGER_NOTE, $debugger));
        $this->io->writeln(sprintf(self::RUNNING_WITH_DEBUGGER_NOTE, $debugger), OutputInterface::VERBOSITY_QUIET);
    }
```

Reply:
@sanmai yeah sure. Thought about it. 

but symfony will do it a bit later here https://github.com/symfony/console/blob/master/Application.php#L845